models/notifications/requests/fcm_request.py

```diff
@@ -0,0 +1,69 @@
+from firebase_admin import messaging
+
+from models.notifications.requests.request import Request
+
+
+class FCMRequest(Request):
+    """ Represents a notification payload and a delivery option to send to FCM.
+    https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages
+    Attributes:
+        notification (Notification): The Notification to send.
+        tokens (list, string): The FCM registration tokens (up to 100) to send a message to.
+    """
+
+    def __init__(self, app, notification, tokens=None):
+        """
+        Note:
+            Should only supply one delivery method - either token, topic, or connection.
+        Args:
+            app (firebase_admin.App): The Firebase app to send to.
+            notification (Notification): The Notification to send.
+            tokens (list, string): The FCM registration tokens (up to 100) to send a message to.
+        """
+        super(FCMRequest, self).__init__(notification=notification)
+
+        self._app = app
+
+        if len(tokens) > 100:
+            raise ValueError('FCMRequest tokens must contain less than 100 tokens')
+
+        self.tokens = tokens
+
+    def __str__(self):
+        return 'FCMRequest(tokens={}, notification={})'.format(self.tokens, str(self.notification))
+
+    def send(self):
+        """ Attempt to send the notification.
+        Returns:
+            messaging.BatchResponse - Batch response object for the messages sent.
+        """
+        from sitevars.notifications_enable import NotificationsEnable
```

Comment:
This depends on https://github.com/the-blue-alliance/the-blue-alliance/pull/2579 🙇